lib/pipeline/statelessPipelineStack.ts

```diff
@@ -20,6 +20,41 @@ export class StatelessPipelineStack extends cdk.Stack {
   constructor(scope: Construct, id: string, props: cdk.StackProps) {
     super(scope, id, props);
 
+    // Define CodeBuild project for GH action runner to use
+    // the GH repo defined below already configured to allow CB webhook
+    // This is actually not part of the pipeline, so I guess we could move this someday.
+    const ghRunnerRole = new iam.Role(this, 'GHRunnerRole', {
+      assumedBy: new iam.ServicePrincipal('codebuild.amazonaws.com'),
+    });
+    new codebuild.CfnProject(this, 'GHRunnerCodeBuildProject', {
```

Comment:
Well, this webhook GH runner <> AWS CodeBuild is [new](https://aws.amazon.com/about-aws/whats-new/2024/04/aws-codebuild-managed-github-action-runners/), so is not a surprised that the webhook flag for L2 CDK construct hasn't exist yet. Defining here the L1 construct.

[cdk-issue-#30062](https://www.github.com/aws/aws-cdk/issues/30062)